js/data.js

```diff
@@ -6,3 +6,10 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
+window.addEventListener('beforeunload', function (event) {
+  var entriesJSON = JSON.stringify(data.entries);
+  this.localStorage.setItem('entries', entriesJSON);
```

Comment:
You should not save the `entries` individually, they are already saved in the entire data model in `data`.